php/Blockonomics.php

```diff
@@ -215,12 +209,6 @@ public function examine_server_callback_urls($response_body, $crypto)
 
     public function check_callback_urls_or_set_one($crypto, $response) 
     {
-        $api_key = get_option("blockonomics_api_key");
```

Comment:
This was placed in this location to allow [fetching of callback urls using the temp_api_key](https://github.com/blockonomics/woocommerce-plugin/blob/master/php/Blockonomics.php#L367-L368), but since this functionality has been removed, it makes sense to 1st check the API key before making any requests or running test setup for each crypto:
https://github.com/blockonomics/woocommerce-plugin/pull/371/files#diff-a3307e680eb76b9d93556c9ae2e5dc134b3526246480730fb8ddf49186da29ebR308-R313

Reply:
Moved into each crypto error message to use existing setup error display (Reduce changes in this PR): https://github.com/blockonomics/woocommerce-plugin/pull/371/files#diff-a3307e680eb76b9d93556c9ae2e5dc134b3526246480730fb8ddf49186da29ebR316-R321